cfn-resources/test/e2e/utility/atlas_helper.go

```diff
@@ -32,47 +31,38 @@ type AtlasEnvOptions struct {
 	BaseURL    string
 }
 
-const (
-	cfnTool = "mongodbatlas-cloudformation-resources"
-)
-
-var (
-	toolName  = cfnTool
-	userAgent = fmt.Sprintf("%s/%s (%s;%s)", toolName, version.Version, runtime.GOOS, runtime.GOARCH)
-)
```

Comment:
was used for creating instance of old sdk used for testing, this was replaces with c.NewSDKV2Client which defines the userAgent within the function.